### PR TITLE
New endpoints/rows 102 114

### PR DESCRIPTION
**Pull request by morgan-sl**

### CODE REVIEW

File: pipeline/lookup.csv [hunk 1]
```diff
@@ -29678,3 +29678,921 @@ brownfield-land,,local-authority-eng:BAE,BFR53,1729669
...
+brownfield-land,,local-authority-eng:ERE,BLR63,1730554
+brownfield-land,,local-authority-eng:FAR,3247,1730555
+brownfield-land,,local-authority-eng:FAR,3253,1730556
+brownfield-land,,local-authority-eng:FAR,3255,1730557
+brownfield-land,,local-authority-eng:FAR,3257,1730558
+brownfield-land,,local-authority-eng:FAR,3259,1730559
+brownfield-land,,local-authority-eng:FAR,3260,1730560
+brownfield-land,,local-authority-eng:FAR,3263,1730561
+brownfield-land,,local-authority-eng:FAR,3264,1730562
+brownfield-land,,,,1730563
```
Comment (eveleighoj) [inline pipeline/lookup.csv]: These lines are incorrect, they cannot be blank for organisation and reference

File: pipeline/lookup.csv [hunk 2]
```diff
@@ -29678,3 +29678,921 @@ brownfield-land,,local-authority-eng:BAE,BFR53,1729669
...
+brownfield-land,,local-authority-eng:EPP,B37,1730480
+brownfield-land,,local-authority-eng:EPP,B39,1730481
+brownfield-land,,local-authority-eng:EPP,B40,1730482
+brownfield-land,,local-authority-eng:EPP,B42,1730483
+brownfield-land,,local-authority-eng:EPP,B43,1730484
+brownfield-land,,local-authority-eng:EPP,B44,1730485
+brownfield-land,,local-authority-eng:EPP,B45,1730486
+brownfield-land,,local-authority-eng:EPP,B46,1730487
+brownfield-land,,local-authority-eng:EPP,B47,1730488
+brownfield-land,,,B50,1730489
```
Comment (eveleighoj) [inline pipeline/lookup.csv:30497]: No organisation here so it is incorrect

File: pipeline/lookup.csv (hunk 1 above)
Comment (morgan-sl) [reply pipeline/lookup.csv]: I've removed the incorrect lines from lookup.csv in change 102-114 linked to this PR request

File: pipeline/lookup.csv (hunk 2 above)
Comment (morgan-sl) [reply pipeline/lookup.csv:30497]: I've removed the incorrect lines from lookup.csv in change 102-114 linked to this PR request

File: pipeline/lookup.csv [hunk 3]
```diff
@@ -29678,3 +29678,892 @@ brownfield-land,,local-authority-eng:BAE,BFR53,1729669
...
+brownfield-land,,local-authority-eng:BRD,SW/157,1729834
+brownfield-land,,local-authority-eng:BRD,SW/158,1729835
+brownfield-land,,local-authority-eng:BRD,SW/160,1729836
+brownfield-land,,local-authority-eng:BRD,SW/163,1729837
+brownfield-land,,local-authority-eng:BRD,SW/165,1729838
+brownfield-land,,local-authority-eng:BRD,SW/168,1729839
+brownfield-land,,local-authority-eng:BRD,SW/170,1729840
+brownfield-land,,local-authority-eng:BRD,TH/037,1729841
+brownfield-land,,local-authority-eng:BRD,WI/002,1729842
+brownfield-land,,local-authority-eng:BRC,Map Ref 12 BRA7 Eastern Gateway Development Area, Bracknell (Town Square) (Submission Local Plan),1729843
```
Comment (eveleighoj) [inline pipeline/lookup.csv:29851]: This and any other lookup where a reference has a comma in it need quotation marks

File: pipeline/lookup.csv [hunk 4]
```diff
@@ -29678,3 +29678,892 @@ brownfield-land,,local-authority-eng:BAE,BFR53,1729669
...
+brownfield-land,,local-authority-eng:BRC,Map Ref 14 BRA12 Former Bus Depot (Submission Local Plan),1729844
+brownfield-land,,local-authority-eng:BRC,Map Ref 43 BRA 17 Bus Station (part of the Southern Gateway Development Area, Bracknell) (Submission Local Plan),1729845
+brownfield-land,,local-authority-eng:BRC,Map Ref 55,1729846
+brownfield-land,,local-authority-eng:BRC,Map ref 57 LP8 BRA18 The Peel Centre and The Point (Submission Local Plan),1729847
+brownfield-land,,local-authority-eng:BRC,Map ref 58,1729848
+brownfield-land,,local-authority-eng:BRC,Map ref 59,1729849
+brownfield-land,,local-authority-eng:BRC,Map ref 60,1729850
+brownfield-land,,local-authority-eng:BRC,Map ref 61,1729851
+brownfield-land,,,Map ref 62,1729852
```
Comment (eveleighoj) [inline pipeline/lookup.csv:29860]: These two also don't have an organisation, why is this? It seems to have the same style of reference as above but no org?

File: pipeline/lookup.csv [hunk 5]
```diff
@@ -29678,3 +29678,892 @@ brownfield-land,,local-authority-eng:BAE,BFR53,1729669
...
+brownfield-land,,local-authority-eng:CAT,BR-124,1729858
+brownfield-land,,local-authority-eng:CAT,BR-125,1729859
+brownfield-land,,local-authority-eng:CAT,BR-127,1729860
+brownfield-land,,local-authority-eng:CAT,BR-128,1729861
+brownfield-land,,local-authority-eng:CAT,BR-129,1729862
+brownfield-land,,local-authority-eng:CAT,BR-117,1729863
+brownfield-land,,local-authority-eng:CAT,BR-118,1729864
+brownfield-land,,local-authority-eng:CAT,BR-120,1729865
+brownfield-land,,local-authority-eng:CAT,BR-126,1729866
+brownfield-land,,local-authority-eng:CAR,,1729867
```
Comment (eveleighoj) [inline pipeline/lookup.csv:29875]: This and rows below have no reference this shouldn't be in the lookup and it's weird that it's happening.

File: pipeline/lookup.csv [hunk 6]
```diff
@@ -29678,3 +29678,892 @@ brownfield-land,,local-authority-eng:BAE,BFR53,1729669
...
+brownfield-land,,local-authority-eng:CON,BR271,1730127
+brownfield-land,,local-authority-eng:CON,BR272,1730128
+brownfield-land,,local-authority-eng:CON,BR273,1730129
+brownfield-land,,local-authority-eng:CON,BR274,1730130
+brownfield-land,,local-authority-eng:CON,BR275,1730131
+brownfield-land,,local-authority-eng:CON,BR276,1730132
+brownfield-land,,local-authority-eng:CON,BR277,1730133
+brownfield-land,,local-authority-eng:CON,BR278,1730134
+brownfield-land,,local-authority-eng:CON,BR279,1730135
+brownfield-land,,,CBCBLR044,1730136
```
Comment (eveleighoj) [inline pipeline/lookup.csv:30144]: No organisation on these lookups?

File: pipeline/lookup.csv [hunk 7]
```diff
@@ -29678,3 +29678,892 @@ brownfield-land,,local-authority-eng:BAE,BFR53,1729669
...
+brownfield-land,,local-authority-eng:EAT,BLR-9-5,1730390
+brownfield-land,,local-authority-eng:EAT,BLR-11-1,1730391
+brownfield-land,,local-authority-eng:EAT,BLR-12-1,1730392
+brownfield-land,,local-authority-eng:EAT,BLR-3-7,1730393
+brownfield-land,,local-authority-eng:EAT,BLR-4-13,1730394
+brownfield-land,,local-authority-eng:EAT,BLR-4-14,1730395
+brownfield-land,,local-authority-eng:EAT,BLR-10-1,1730396
+brownfield-land,,local-authority-eng:EDE,BL39,1730397
+brownfield-land,,local-authority-eng:EDE,BL40,1730398
+brownfield-land,,,BLR0302,1730399
```
Comment (eveleighoj) [inline pipeline/lookup.csv:30407]: No organisations

**Review by eveleighoj — COMMENTED** [review]
Some additional changes that need to be made. It's odd that there are so many with organisation missing

File: pipeline/lookup.csv (hunk 7 above)
Comment (morgan-sl) [reply pipeline/lookup.csv:30407]: I've removed entries with no organisation, and quoted a couple more referencence fields containing commas.

Comment (morgan-sl) [reply pipeline/lookup.csv:30407]: rolled back last 2 check-ins and applied corrections from 1729345.
This is the first entry George applied, whom I believe was the first to push changes from this list.

**Review by eveleighoj — APPROVED** [review]

**Pull request merged**